### DISCUSSION TITLE
Farming on a virtual machine like DigitalOcean

### DISCUSSION THREAD

**Discussion by 2smpl2** [Chia-Network/chia-blockchain#552] [Ideas]
Hey all. As the title suggests, has anyone had any luck setting up farming Chia on a virtual machine like DigitalOcean? If so, please could you share some tips on how to get started?

I am limited by my local resources so I thought I would explore this idea.

Thanks in advance!

**Comment by xorinox** [comment]
I would think that DigitalOcean provides Linux VMs as any other cloud provider, so installing as described in https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL   should be just fine.

The much larger question is, how much are willing to pay for storage of Chia plots? The minimum required plot size for main-net is going to be K32 which is about 101 GiB per file. I doubt that with any cloud based provider profitable farming can be done.

**Comment by BrandtH22** [comment]
It is generally not recommended to farm on cloud storage solutions due to increased latency. 
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
